features/install_uninstall.feature

```diff
@@ -80,17 +81,8 @@ Feature: UA Install and Uninstall related tests
         """
         Then I verify that running `dpkg-reconfigure ubuntu-advantage-tools` `with sudo` exits `0`
 
-        # Works even when user overwrites /usr/bin/python3 with their version
-        When I run `ln -sf /usr/local/bin/python3.10 /usr/bin/python3` with sudo
-        Then I verify that running `/usr/bin/python3 -c "import uaclient"` `with sudo` exits `1`
-        Then stderr matches regexp:
-        """
-        No module named 'uaclient'
-        """
-        Then I verify that running `dpkg-reconfigure ubuntu-advantage-tools` `with sudo` exits `0`
-
         Examples: ubuntu release
            | release | deadsnakes-pkg | deadsnakes-version |
            | xenial  | python3.9      | 3.9.4              |
            | bionic  | python3.9      | 3.9.7              |
-           | focal   | python3.9      | 3.9.7              |
\ No newline at end of file
+           | focal   | python3.9      | 3.9.7              |
```

Comment:
Wow totally my bad. Can you remove the deadsnakes columns while we're here? I was originally going to use that ppa for the test, but that didn't work out and I accidentally left these here (deadsnakes ppa builds are properly configured to use dist-packages)